### DISCUSSION TITLE
ProHd: Bricked, Unbricked, Bricked again 7 month later

### DISCUSSION THREAD

**Discussion by bradut** [GerryDazoo/Slinger#632] [Q&A]
Hi, 
My SlingBox got bricked again after 7 month of working fine.
Nothing has changed in its setup :
- the local IP is the same
- the SlingBox is still listening to its assigned port
- the outgoing port 5678 from the SlingBox is still blocked. (I also blocked a lot of other ports, just in case...)

This issue had happened before: initially it got bricked in just two weeks, but since 20 March 2023 it was OK (see [this thread](https://github.com/GerryDazoo/Slinger/discussions/525)).

**Can anyone please spot any issue with my settings?**
<br>

**These are the router settings:**

![image](https://github.com/GerryDazoo/Slinger/assets/2496596/f606517f-8402-4f94-a8d2-71eaf4583f70)
Adding a rule to block outgoing port 5678
<br>

![image](https://github.com/GerryDazoo/Slinger/assets/2496596/fcc81240-b8ec-4934-b023-cdc97a66d681)
List of blocked ports
<br>

![image](https://github.com/GerryDazoo/Slinger/assets/2496596/00619b4e-941f-4473-bb6a-237de1bbc6c6)
Slinger identifies that the SlingBox might be bricked
<br>

![image](https://github.com/GerryDazoo/Slinger/assets/2496596/20f4546d-7e4f-45c9-bde4-6c41cd7340e6)
Slinger is right: SlingPlayer also indicates that the SlingBox is bricked:
Operation failed
Error:0x92340070

**Comment by flipper2222222** [comment]
not sure why you box got bricked but personally I block all internet access to the slingbox by MAC address

**Comment by bradut** [comment]
I believe the reason could be a combination of a 'temperamental' SlingBox and an unreliable router. 

The SlingBox has/had a few issues:
- It ignores its assigned static IP.
- Before being bricked, it ignored Ch+/Ch- commands, whether sent from SlingPlayer (1.5.x or 2.x) or Slinger. Now it works...

So, although highly improbable, maybe the SlingBox temporarily took another IP, which wasn't protected against invoking the Kraken at port 5678. 
I've now blocked that port for any device connected to that router.
